src/translib/transformer/xfmr_vrf.go

```diff
@@ -21,12 +22,12 @@ const (
         MGMT_VRF_NAME          = "mgmt-vrf-name"
 )
 
+const (
+        DEFAULT_NETWORK_INSTANCE_CONFIG_TYPE        = "L3VRF"
+)
+
 var nwInstTypeMap = map[ocbinds.E_OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE] string {
-        ocbinds.OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE_UNSET: "",
         ocbinds.OpenconfigNetworkInstanceTypes_NETWORK_INSTANCE_TYPE_DEFAULT_INSTANCE: "DEFAULT_INSTANCE",
```

Comment:
other than "default" VRF name, if user sets type "DEFAULT_INSTANCE", can we throw an error?